src/java.base/share/classes/java/util/zip/ZipFile.java

```diff
@@ -1869,15 +1869,15 @@ private EntryPos getEntryPos(String name, boolean addSlash) {
 
                     // Compare the lookup name with the name encoded in the CEN
                     switch (zc.compare(name, cen, noff, nlen, addSlash)) {
-                        case EXACT_MATCH:
+                        case ZipCoder.EXACT_MATCH:
```

Comment:
This would perhaps read cleaner with a static import of the constants?

Reply:
No strong feeling about this, but I lean towards not using static imports unless there's repeated use.

Reply:
No strong feelings here either, just thought it made the code easier on the eye. I'll leave it to your preference.